staging/src/k8s.io/apiserver/pkg/util/logs/logs.go

```diff
@@ -69,8 +69,12 @@ func NewLogger(prefix string) *log.Logger {
 	return log.New(GlogWriter{}, prefix, 0)
 }
 
-// GlogSetter is a setter to set glog level.
-func GlogSetter(val string) (string, error) {
+// GlogSetterOrGetter is to used to set glog level or get glog level .
+func GlogSetterOrGetter(val string) (string, error) {
```

Comment:
mixing responsibilities in a single function like this isn't good... setting a flag to an empty string could be a reasonable thing to do. using it as a signal we want a get instead isn't great